docs/charge/how-to-create-charge-woovi-parcelado.mdx

```diff
@@ -0,0 +1,148 @@
+---
+id: how-to-create-charge-woovi-parcelado
+title: Como usar a API para criar uma cobrança com OpenPix Parcelado?
+tags:
+  - concept
+  - api
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+:::info
+Para a ultilização dessa funcionalidade é necessário possuir a funcionalidade OpenPix Parcelado. Solicite o acesso via chat pela plataforma.
+:::
```

Comment:
dispose this link?
https://app.woovi.com/home/charges/pix-credit-card?begin=2023-08-01T03%3A00%3A00.000Z&end=2023-08-03T02%3A59%3A59.999Z

Reply:
sure, nice catch